engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/ImportsAndFQNConsistencyTest.java

```diff
@@ -40,11 +39,18 @@
  */
 @RunWith(Parameterized.class)
 public class ImportsAndFQNConsistencyTest {
-  private static Context ctx;
 
   /** Used for description in {@link PrintCodeRule} test rule. */
   private static String code;
 
+  @ClassRule
+  public static final ContextRule ctxRule =
+      ContextRule.createCustom(
+          () ->
```

Comment:
When designing new API we could choose a better alternative than `Supplier<Context>`. There is [concept of context modifiers](https://github.com/enso-org/enso/blob/8404e00a7464b953c52ced05a413278ff65b6e41/engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/InterpreterTest.scala#L332) in `InterpreterTest.

That could make it:
```java
modifyContext((ctx) -> ctx.option(RuntimeOptions.DISABLE_IR_CACHES, "false"))
```

Reply:
Implemented via [ContextRule.Builder.withModifiedContext](https://github.com/enso-org/enso/blob/c39474106a5d4ae5204fed10fbfaab62374fd2d9/lib/java/test-utils/src/main/java/org/enso/test/utils/ContextRule.java#L214)